src/TLuaInterpreterMapper.cpp

```diff
@@ -3637,6 +3624,7 @@ int TLuaInterpreter::setRoomIDbyHash(lua_State* L)
     }
     host.mpMap->mpRoomDB->hashToRoomID[hash] = id;
     host.mpMap->mpRoomDB->roomIDToHash[id] = hash;
+    host.mpMap->update();
```

Comment:
This function does not modify the visible appearance of the map so I don't think this useful here.

Reply:
Updated.